lib/cms/repo.ex

```diff
@@ -343,17 +410,17 @@ defmodule CMS.Repo do
 
   @doc "Get all the events, paginating through results if needed, and caches the result"
   @spec events_for_year(Calendar.year()) :: [%Teaser{}]
+  @decorate cacheable(cache: @cache, on_error: :nothing, opts: [ttl: @ttl])
```

Comment:
Not sure this is needed since you also decorate the `do_events_for_range/3` function